src/rust/logger/src/sender.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use crate::*;
+
+#[derive(Clone)]
+pub struct LogSender {
+    pub(crate) debug: Arc<dyn Log>,
```

Comment:
The thing that bugs me slightly about this struct is that level is only applicable to debug, and command needs a different set of options (for now, sampling rate). Can we bundle these sender specific configs with the corresponding Log instead of listing them side by side?